README.md

```diff
@@ -30,6 +30,7 @@ Allows you to run [Apex Static Analysis](http://pmd.sourceforge.net/snapshot/pmd
 - `priorityWarnThreshold`: Determines at what priority level 'warnings' will be added. Anything less will be a hint
 - `enableCache`: Creates a cache file for PMD to run faster. Will create a .pmdCache file in your workspace
 - `pmdBinPath` (prev. `pmdPath`) (optional): set to override the default pmd binaries. This should point to the PMD folder which contains folders `lib` and `bin`. Most likely it is called `libexec`.
+- `additionalClassPaths` (optional): set of paths to be appended to classpath. Used to find custom rule definitions. Can be absolute or relative to workspace.
```

Comment:
We might want to include a link to the [PMD Documentation for how to define custom rules](https://pmd.github.io/latest/pmd_userdocs_extending_writing_pmd_rules.html)

Reply:
> Used to find custom rule definitions 

Should this be:

Used to **define** custom rule definitions

Reply:
I've updated the readme and setting description.

It's not defining the custom rules, that is done in the java source, but rather is used to find the jar file containing those definitions.

Reply:
@andrewgilbertsagecom  yea I understand.  Do you happen to have a simple jar that you could share?  Ideally we would have a simple "hello world" use case that we could use to make sure this feature works (maybe even unit test).

Reply:
Disregard that last comment! should have kept reading! 

Going to try and run it with your simple rule.  Thanks!